pkg/MoneyAccount/MoneyAccount.go

```diff
@@ -0,0 +1,64 @@
+package MoneyAccount
+
+import (
+	"time"
+)
+
+type Cuenta interface {
+	GetId() string
+}
+
+type Account struct {
+	Nombre      string
+	saldo       float64
```

Comment:
Por *separation of concerns*, tenéis que distinguir los datos y metadatos de una cuenta del estado de la cuenta en un momento determinado. Y esto es un ejemplo del problema que podríais plantear en el issue correspondiente.